charts/postgres/templates/storage.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 spec:
   storageClassName: manual
   capacity:
-    storage: 5Gi
+    storage: 10Mi
```

Comment:
This may be a bit small? Would prefer not to have to deal with volume scaling scenarios during the period we are running with the local postgres, before switching to an RDS postgres